src/common/FilteredLigandResults.js

```diff
@@ -159,9 +174,9 @@ FilteredLigandResults.propTypes = {
   dockHandler: propTypes.func,
   /** The handler for selecting a  */
   handleSelectedResult: propTypes.func,
-  /** An object representing a ligand selected to be viewed 
```

Comment:
Can we include these semantic corrections in other commits/pull requests? It's not like we need to review these.